docs/conf.py

```diff
@@ -48,9 +48,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '19.12'
+version = '20.01'
 # The full version, including alpha/beta/rc tags.
-release = '19.12.0-edge'
+release = '20.01.0-SNAPSHOT'
 
```

Comment:
Please do not commit changes to this file

Reply:
removed from commit by
commit d94d05db3724e3e28605d2268193e209eb0d32d8